src/main/java/org/spongepowered/common/command/SpongeCommandManager.java

```diff
@@ -67,9 +68,6 @@
 import java.util.Set;
 import java.util.function.Function;
 
-import javax.annotation.Nullable;
-import javax.inject.Inject;
```

Comment:
`javax` should come after `java`, keep these here
Remember - https://docs.spongepowered.org/en/contributing/implementation/codestyle.html